tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -265,21 +265,21 @@ exports[`show detailed usage with --help no-semi (write) 1`] = `Array []`;
 exports[`show detailed usage with --help parser (stderr) 1`] = `""`;
 
 exports[`show detailed usage with --help parser (stdout) 1`] = `
-"--parser <flow|babylon|typescript|css|less|scss|json|graphql|markdown>
+"--parser <babylon|css|flow|graphql|json|less|markdown|scss|typescript>
```

Comment:
There was a logical grouping here that has broken

Reply:
I can see that there are more errors and there are some trailing spaces that were removed..  do we really need those?